highfidelity/hifi
# Fix simulation owned entities not to go to origin after cache reload (+ small priority fix)

Fixes bug where the user is still in the same session while having simulation ownership over some entities but the entity tree is reset, this caused the entities to show up at origin of domain before this PR.

## QA Testing

- Have at least three interface controlled avatars in this test
- Physics should work as smooth as they work in the master branch
- Selecting an entity in edit mode followed by reloading the cache should not move the last selected entity to the origin anymore.
- Play around with the physics, reconnect a couple of times, reload cache a couple of times, make sure it all works like usual.
